tutorials/beginner/rulesscripts.md

```diff
@@ -1,49 +0,0 @@
----
-layout: tutorial-beginner
----
-
-{% include base.html %}
-
-# Working with rules and scripts
-
-This part is not yet written.
-
-
----
-
-*Guidelines*
-
-*I think this will become the longest section*
-
-*Don't forget to address JSR233 or other alternatives to the DSL that will be built into OH 2*
-
-* *Event driven development*
-
-* *Format and structure of rules*
-
-* *Difference between actions and posting sending things to items*
-
-* *Functions*
-
-* *Group.members manipulation and how it can help simplify your rules*
-
-* *How to figure out what to import*
-
-* *Working with Time*
-
-* *Working with Timers*
-
-* *Casting to appropriate types and when that is necessary*
-
-* *Working with Numbers*
-
-* *Working with States*
-
-* *Lambdas -> in advanced openHAB*
-
-* *...*
-
```

Comment:
Did you make sure this list is available somewhere else? (community forum documentation for example)